ansible/roles/baselayout/vars/main.yml

```diff
@@ -88,7 +98,8 @@ packages: {
     'gccmakedep',
     'git',
     'gmake',
-    'xz'
+    'xz',
+    'sudo',
```

Comment:
Just wondering why we need this now and we did not before ?

Reply:
It used to be in the always install section, but you can't brew install it so now it's moved to all the existing ones.

Reply:
got it